extensions/liquibase/deployment/src/main/java/io/quarkus/liquibase/LiquibaseProcessor.java

```diff
@@ -187,11 +188,11 @@ ServiceStartBuildItem configureRuntimeProperties(LiquibaseRecorder recorder,
     private void addService(BuildProducer<ReflectiveClassBuildItem> reflective,
             BuildProducer<GeneratedResourceBuildItem> generatedResourceProducer,
             BuildProducer<NativeImageResourceBuildItem> resourceProducer,
-            Class<?> className) {
+            Class<?> className, boolean methods) {
 
         Class<?>[] impl = ServiceLocator.getInstance().findClasses(className);
         if (impl != null && impl.length > 0) {
-            reflective.produce(new ReflectiveClassBuildItem(true, false, false, impl));
+            reflective.produce(new ReflectiveClassBuildItem(true, methods, false, impl));
```

Comment:
I think I would make it true for all services.

Reply:
It was my first solution, but I'm getting errors for some classes at native compile time.
I'll try again and create two class lists.

Does it make big different for the native image (compile time) if we register all the classes?

Reply:
Done, only the `LicenseService` class needs to be `methods=false`

Reply:
So doing that might include more methods than strictly needed in the native image but I think it's safer.

As for the `LicenseService` it's probably related to the substitutions you had to make there so looks OK to me.